package.json

```diff
@@ -2,7 +2,7 @@
   "name": "vscode-knative",
   "displayName": "Knative",
   "description": "Knative extension for VSCode",
-  "version": "0.9.4",
+  "version": "1.0.0",
   "license": "MIT",
   "publisher": "redhat",
   "preview": true,
```

Comment:
should we remove the preview tag?

Reply:
@mohitsuman

Reply:
For serverless, let us do a few more releases and then move out of the `Preview` tag.